types/sinon/ts3.1/sinon-tests.ts

```diff
@@ -1,4 +1,5 @@
 import sinon = require("sinon");
+import Bluebird = require("bluebird");
```

Comment:
I assumed that pulling Bluebird as a type dependency is not a big deal, since it is already present in the repo.

I can see you point though, so I'll submit a fix in a couple of days.